.github/workflows/Windows-pack.yml

```
@@ -200,7 +200,7 @@ jobs:
           echo $(python $GITHUB_WORKSPACE/scripts/upload_services/transferwee.py upload $GITHUB_WORKSPACE/build/Package/portable/flameshot-${VERSION}-${{ matrix.config.pak_arch }}.zip)
           echo "=====no operation for you can see link in the log console====="
       - name: Artifact Upload
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
-          name: Windows-artifact
+          name: Windows-${{ matrix.config.arch }}-${{ matrix.type }}-${{ matrix.runs-on }}-artifact
```

```suggestion
          name: Windows-${{ matrix.config.arch }}-${{ matrix.type }}-artifact
```

Fix double `-` in name.